utils.js

```diff
@@ -353,8 +353,12 @@ const createCustomMethodHandler = (instance, method) => {
     instance.context.debug(
       `Attempting to run method "${method}" on template aliases: ${components.join(', ')}`
     )
+
+    // Make sure the template is an object
```

Comment:
Why is it important to make sure the template is an object? Can this be explained in this comment?

Reply:
On line 349 above, is it prudent to throw an Error? Perhaps if there was a proper testing framework that would guarantee the behavior of this, however since there is no framework and this is a core part of serverless components, I suggest not throwing an error.

Reply:
`template` input of Template component can accept both `string` (path to file) and an object, which you can prepare in advance and simply pass for execution.